src/main/java/frc/robot/commands/RunIntake.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+/*
+ * Asimov's Laws:
+ * The First Law: A robot may not injure a human being or, through inaction, allow a human being to come to harm.
+ * The Second Law: A robot must obey the orders given it by human beings except where such orders would conflict with the First Law.
+ * The Third Law: A robot must protect its own existence as long as such protection does not conflict with the First or Second Law.
+ */
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.Intake;
+
+public class RunIntake extends Command {
+
+  private Intake intake;
+
+  /** Creates a new RunIntake. */
+  public RunIntake() {
+    this.intake = RobotContainer.intake;
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    intake.runIntake();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {}
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    intake.stop();
+  }
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
+    return false;
```

Comment:
We have two options here. Either return the state of the limit switch from this method, or (better yet) we create this command and a command that ends when the limit switch is `true`, and put them in a ParallelRaceGroup. You need to try the ParallelRaceGroup method first and see if it causes an error where the commands conflict with each other—simulation mode is your bestie (or maybe you can talk electrical into plugging a Spark Max and a limit switch into the test board for you, idk).

Reply:
Okay, decision has been made: do the first option. Just return the state of the limit switch from the `isFinished()` method of `RunIntake`.